include/swift/AST/ASTBridgingWrappers.def

```diff
@@ -31,9 +31,8 @@
 #define AST_BRIDGING_WRAPPER_NONNULL(Name) AST_BRIDGING_WRAPPER(Name)
 #endif
 
-// Each AST node subclass stores a non-null raw value.
 #ifndef DECL
-#define DECL(Id, Parent) AST_BRIDGING_WRAPPER_NONNULL(Id##Decl)
+#define DECL(Id, Parent) AST_BRIDGING_WRAPPER_NULLABLE(Id##Decl)
```

Comment:
At this point I kind of wonder whether we're really gaining much from defining separate macros for can-be-nullable, and never-nullable, and should just stamp out nullable bridging wrappers for everything. My main worry with that initially was that it would unnecessarily pollute the type namespace, but at this point a good chunk of types need nullable wrappers anyways. In any case, I can explore that in a follow up.